sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -212,6 +212,9 @@ statement
     | (DESC | DESCRIBE) TABLE? option=(EXTENDED | FORMATTED)?
         multipartIdentifier partitionSpec? describeColName?            #describeTable
     | (DESC | DESCRIBE) QUERY? query                                   #describeQuery
+    | COMMENT ON (database | NAMESPACE) multipartIdentifier IS
```

Comment:
not related to this PR, but I have seen (database | NAMESPACE) too many times.

Maybe we should have a
```
namespace: DATABASE | SCHEME | NAMESPACE
```

and use `namespace` directly.

Reply:
I created https://github.com/apache/spark/pull/27027 for this.

Reply:
thanks! merged